volatility/framework/plugins/windows/pslist.py

```diff
@@ -183,17 +187,18 @@ def _generator(self):
             else:
                 (_, _, offset, _, _) = list(memory.mapping(offset = proc.vol.offset, length = 0))[0]
 
-            dumped = False
+            file_output = "Disabled"
             if self.config['dump']:
-                filedata = self.process_dump(self.context, self.config['nt_symbols'], pe_table_name, proc)
-                if filedata:
-                    dumped = True
-                    self.produce_file(filedata)
+                file_handle = self.process_dump(self.context, self.config['nt_symbols'], pe_table_name, proc,
+                                                self._file_handler)
```

Comment:
I don't quite follow how a plugin writer would know to use this idiom. What is `self._file_handler`? The underscore has an ominous internal feel to it and I'm having trouble finding how this is defined or documented.